fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/QrFragment.kt

```diff
@@ -44,6 +47,16 @@ class QrFragment : Fragment() {
     return binding.root
   }
 
+  override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+    super.onViewCreated(view, savedInstanceState)
+    setupCopyButton()
+  }
+
+  private fun setupCopyButton() {
+    clipboardManager.setupCopyButton(
+      binding.copyServerButton, binding.pkText, "Public Key")
```

Comment:
Maybe `copyServerButton` should be renamed in `copyPublicKeyButton` or something like that